shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -107,3 +130,133 @@
 }
 
 }  // namespace shell
+
+// This recognizers delays touch events from being dispatched to the responder chain until it failed
+// recognizing a gesture.
+//
+// We only fail this recognizer when asked to do so by the Flutter framework (which does so by
+// invoking an acceptGesture method on the platform_views channel). And this is how we allow the
+// Flutter framework to delay or prevent the embedded view from getting a touch sequence.
+@interface DelayingGestureRecognizer : UIGestureRecognizer <UIGestureRecognizerDelegate>
```

Comment:
Nit: Prefix the names of all private iOS classes with the `Flutter` prefix.

Reply:
done